test/core/templates/options-test.js

```diff
@@ -0,0 +1,59 @@
+const test = require('tape'),
+  base64Img = require('base64-img'),
+  looksSame = require('looks-same');
+
+const ImageSequencer = require('../../../src/ImageSequencer');
+
+const red =
+  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==';
+target = 'test_outputs';
+
+/**
+ * @method optionsChangeTest
+ * @description a common test for modules
```

Comment:
Please capitalize the first letter and add a full stop at the end of a comment.

Reply:
Done!